cmd/sslcheck/main.go

```diff
@@ -51,15 +55,20 @@ func sslcheckCmd() *cobra.Command {
 				configkit.BindFlag("ssl.connectTimeout", cmd.Flag("timeout")),
 				configkit.BindFlag("ssl.warnValidity", cmd.Flag("warning")),
 				configkit.BindFlag("ssl.criticalValidity", cmd.Flag("critical")),
+				configkit.BindFlag("slack.enabled", cmd.Flag("slack")),
 			)
 			if err != nil {
 				return err
 			}
+			err = viper.BindEnv("slack.hookUrl")
+			if err != nil {
+				return err
+			}
```

Comment:
Do you need to manually bind this env var? configkit automatically binds env vars for config keys